opensrp-anc/src/main/java/org/smartregister/anc/library/repository/PreviousContactRepository.java

```diff
@@ -320,6 +320,7 @@ public Facts getPreviousContactFacts(String baseEntityId, String contactNo, bool
         String selection = "";
         String orderBy = "MAX("+ ID + ") DESC";
         String[] selectionArgs = null;
+        int contactNumber = Integer.parseInt(contactNo);
```

Comment:
Won't this break if the `contantNo` is not an Interger?

Reply:
I don't think this code was touched. But will have a look and add necessary checks

Reply:
Yah makes sense. That's why we are parsing it to an Integer however, what happens if the string can't be parsed to an Integer? e.g if the string was `test`

Reply:
yes @dubdabasoduba its had the potential crash issue if any string other than the number is passed now its moved under the try catch block of the code